### DISCUSSION TITLE
Creating a Clickable Number Line

### DISCUSSION THREAD

**Discussion by mshastry2022** [jspsych/jsPsych#900] [Q&A]
Hello!

I was just wondering if there was a way to create a dynamic number line, such that a participant can click on a number and then receive some feedback on the same page, and have this done a specific amount of times. Groups of consecutive numbers will lead to one specific type of feedback! Also, if creating a number line is not possible I was thinking if there was a way to make buttons on a number line image, that when clicked on, leads to the feedback on the same page!

Thanks!
Madhulika

**Comment by vijaymarupudi** [comment]
Could you elaborate on what you mean by feedback on the same page?

Have you checked out this plugin?: https://www.jspsych.org/plugins/jspsych-html-slider-response/
